src/integrations/rewriteframes.ts

```diff
@@ -12,17 +12,25 @@ export function createCapacitorRewriteFrames(): Integration {
   const rewriteFrames = new RewriteFrames({
     iteratee: (frame: StackFrame) => {
       if (frame.filename) {
+
+        if (frame.platform === 'java' || frame.platform === 'cocoa') {
+          // Because platform is not required in StackFrame type
+          // we assume that if not set it's javascript
+          return frame;
+        }
```

Comment:
Does this apply in Capacitor?

In React Native this comes from https://github.com/getsentry/sentry-react-native/blob/66cd5b29cd61243789bad98b33f1364df1681510/src/js/integrations/nativelinkederrors.ts#L31

Reply:
Capacitor doesn't have Native Linked errors so I'll remove this code.